### PR TITLE
fix: added Podman Desktop to the **Development** menu in the Flatpak package

### LINKED ISSUE #3880: File the flatpak application under "Development" menu
### Is your enhancement related to a problem? Please describe

We consider Podman Desktop as a developer tool, but is it filed under the "Utilities" menu, and missing under the "Development" menu.

Flatpak user reported: "The menu system doesn't exhibit a link to this flatpak, it's hard to run without searching for it".

### Describe the solution you'd like

File the flatpak application under "Development" menu as well as "Utilities".

### Describe alternatives you've considered

_No response_

### Additional context

Findability as seen in Discover:

Screenshot_Plasma_20230913_101058

User review as seen in Discover:

Screenshot_Search: podman - 3 items — Discover_20230913_101542

### PR DESCRIPTION
**Pull request by themr0c**
In the Flatpak package, added Podman Desktop to the **Development** category menu, rather then the default **Utilities** category.

fixes #3880 -

Fixes build error:

```
  • application Linux category is set to default "Utility"  reason=linux.category is not set and cannot map from macOS docs=https://www.electron.build/configuration/linux
```

To test (on Linux):

1. Build the flatpak

   ```
   $ MODE=production npm run build && ./node_modules/.bin/electron-builder build --config .electron-builder.config.cjs --linux flatpak
   ```

2. Install the flatpak

   ```
   $ flatpak install -u dist/podman-desktop-1.4.0-next.flatpak
   ```

4. Open the development menu:
 
   Screenshot_Plasma_20230915_071258

### CODE REVIEW

**Comment by benoitf** [comment]
shouldn't it match the categories from the .desktop file ?
https://github.com/containers/podman-desktop/blob/main/.flatpak.desktop#L8

so both Utility and Development ?

**Comment by themr0c** [comment]
The "Utility" menu being the default category, you can find there all applications that forgot to put a proper setup. Therefore, the menu content is a mess, and I believe it is not a feature to stay listed there.

**Comment by benoitf** [comment]
@themr0c but it's a discrepancy with the .desktop file being used on flathub

File: .electron-builder.config.cjs [hunk 1]
```diff
@@ -132,6 +132,7 @@ const config = {
     ],
   },
   linux: {
+    category: 'Development',
```
Comment (themr0c) [inline .electron-builder.config.cjs:135]:
```suggestion
    category: ['Development', 'Utility'],
```

**Comment by mairin** [comment]
I tried to build this to test but it didn't work - 

```
src/extension.ts → dist...
created dist in 1.2s
  • electron-builder  version=23.6.0 os=6.4.14-200.fc38.x86_64
  • loaded configuration  file=/home/duffy/Repositories/podman-desktop/.electron-builder.config.cjs
  • description is missed in the package.json  appPackageFile=/home/duffy/Repositories/podman-desktop/package.json
  • author is missed in the package.json  appPackageFile=/home/duffy/Repositories/podman-desktop/package.json
  ⨯ Invalid configuration object. electron-builder 23.6.0 has been initialized using a configuration object that does not match the API schema.
 - configuration.linux.category should be:
   null | string
   -> The [application category](https://specifications.freedesktop.org/menu-spec/latest/apa.html#main-category-registry).
     How to fix:
     1. Open https://www.electron.build/configuration/linux
     2. Search the option name on the page (or type in into Search to find across the docs).
       * Not found? The option was deprecated or not exists (check spelling).
       * Found? Check that the option in the appropriate place. e.g. "title" only in the "dmg", not in the root.
  failedTask=build stackTrace=ValidationError: Invalid configuration object. electron-builder 23.6.0 has been initialized using a configuration object that does not match the API schema.
 - configuration.linux.category should be:
   null | string
   -> The [application category](https://specifications.freedesktop.org/menu-spec/latest/apa.html#main-category-registry).
     How to fix:
     1. Open https://www.electron.build/configuration/linux
     2. Search the option name on the page (or type in into Search to find across the docs).
       * Not found? The option was deprecated or not exists (check spelling).
       * Found? Check that the option in the appropriate place. e.g. "title" only in the "dmg", not in the root.
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        at validate (/home/duffy/Repositories/podman-desktop/node_modules/@develar/schema-utils/dist/validate.js:86:11)
    at Object.validateConfig (/home/duffy/Repositories/podman-desktop/node_modules/app-builder-lib/src/util/config.ts:240:3)
    at Packager._build (/home/duffy/Repositories/podman-desktop/node_modules/app-builder-lib/src/packager.ts:342:5)
    at Packager.build (/home/duffy/Repositories/podman-desktop/node_modules/app-builder-lib/src/packager.ts:337:12)
    at Object.executeFinally (/home/duffy/Repositories/podman-desktop/node_modules/builder-util/src/promise.ts:12:14)

```

**Comment by mairin** [comment]
(My guess about the ^ above error is that the "Category" target (https://www.electron.build/configuration/linux) is a string, not an array of strings, so you probably can't pass two. We didn't get this error before because we only used utility.

**Comment by benoitf** [comment]
I was wondering if we can use `;` if it's writing the category with `;` separator in the file

**Comment by mairin** [comment]
It is building fine now that I manually removed 

`category: ['Development', 'Utility'],`

and replaced with

`category: 'Development',`

And when I install it, it shows up in the development menu ('Ríomhchlárú') and not utilities:

Screenshot from 2023-09-18 09-10-54

**Comment by benoitf** [comment]
https://github.com/electron-userland/electron-builder/blob/21f3069cb6dcad30959af4bfd8f3014133a3dfde/packages/app-builder-lib/src/targets/LinuxTargetHelper.ts#L189

**Comment by mairin** [comment]
> I was wondering if we can use `;` if it's writing the category with `;` separator in the file

Trying that and rebuilding... so far so good

**Comment by benoitf** [comment]
the idea is that the generated .desktop file should match https://github.com/containers/podman-desktop/blob/main/.flatpak.desktop#L8 that is used for flathub

**Comment by mairin** [comment]
So with the ;, it builds OK, but at least in GNOME, it only appears under "Development" and still does not appear under "Utilities" - is this an issue?

**Comment by themr0c** [comment]
Aligned the .desktop and the electron files to add the app only to the Development category.

Build ran fine.

VS Code tried to reformat entirely the electron file. I discarded the changes and edited the file again with Vim.

**Review by benoitf — APPROVED** [review]

**Pull request merged**